hi-ml-azure/src/health_azure/utils.py

```diff
@@ -608,10 +612,11 @@ def get_workspace(aml_workspace: Optional[Workspace] = None, workspace_config_pa
         else:
             raise ValueError("No workspace config file given, nor can we find one.")
 
-    if workspace_config_path.is_file():
+    if workspace_config_path:
```

Comment:
Why are we removing the check for is_file() here? Before this we already handle the case where it is None, so it makes sense to check that the file exists at this stage.

Reply:
I agree with you. But for a reason I don't understand the is_file() raises an error "AttributeError: 'Workspace' object has no attribute 'is_file'". If I breakpoint and check the type returned workspace_config_path is indeed a Path - and Workspace.from_config(path=str(workspace_config_path), auth=auth) works - and it should not if workspace_config_path is not a path. This is the only solution I found to move forward

Reply:
Do you have repro steps for this? It seems like a different issue than the description of the PR

Reply:
Agreed is a different issue (and I am not fixing it just working around it). I can reproduce the issue - tried a few times - but run out of ideas on how to address. How you suggest to move forward? 

I am primarily trying to work on the DeepMIL PANDA pipeline and this was one of the many things broken - happy to leave this as unmerged branch for now if you think it's not a good enough solution

Reply:
I added this condition `if isinstance(workspace_config_path, Path) and not workspace_config_path.is_file():` to cover for the case a path is passed but the path does not exist

Reply:
Discussed at voice and reverted this change plus added coverage for wrong type in input